docker.go

```diff
@@ -31,11 +32,34 @@ func (d *DockerManager) Start() error {
 	ec := make(chan error)
 	d.docker.StartMonitorEvents(d.eventCallback, ec)
 	go func() {
+		backoff := 1 * time.Second
```

Comment:
Can we use some library for this? https://github.com/cenkalti/backoff seems quite clean.